### ISSUE TITLE
Cannot start containers after docker restart

### ISSUE THREAD

**Issue by devlina** [docker/docker#22483]
Existing containers are not starting up. Everything was working 3 days back. Yesterday when I was creating docker image it failed with error: `error: rpc error: code = 2 desc = \"containerd: container did not start before the specified timeout\""`
without much idea, I just restarted the docker with `service docker restart`. After the restart no containers are starting now. Here is a docker log snippet: 
```
time="2016-05-02T22:45:57.442095707-07:00" level=error msg="Force shutdown daemon"
^[[34mINFO^[[0m[459555] stopping containerd after receiving terminated
Waiting for /var/run/docker.sock
time="2016-05-02T22:45:58.524394884-07:00" level=info msg="New containerd process, pid: 25228\n"
/var/run/docker.sock is up
time="2016-05-02T22:46:09.022715843-07:00" level=info msg="[graphdriver] using prior storage driver \"aufs\""
time="2016-05-02T22:46:14.589655495-07:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
time="2016-05-02T22:46:14.619361391-07:00" level=info msg="Firewalld running: false"
time="2016-05-02T22:46:14.705375286-07:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
time="2016-05-02T22:46:19.365588729-07:00" level=warning msg="Your kernel does not support swap memory limit."
time="2016-05-02T22:46:24.075631365-07:00" level=warning msg="mountpoint for pids not found"
time="2016-05-02T22:46:24.077026168-07:00" level=info msg="Loading containers: start."
............................time="2016-05-02T22:46:24.144734169-07:00" level=warning msg="error locating sandbox id e279e5dcc867b3d220238ae8ba3afd6aed6208d255441399c655e3d52c19255c: sandbox e279e5dcc867b3d220238ae8ba3afd6aed6208d255441399c655e3d52c19255c not found"
time="2016-05-02T22:46:24.145845373-07:00" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/c819b4f7398ac95849d8c388cddcf42cedaaf1041c61be1ff68ba429ec60db0c/shm: invalid argument"
time="2016-05-02T22:46:24.156859717-07:00" level=warning msg="error locating sandbox id cd2b06716a4aece8d92d34558e4106334eb7f2b7ff96d6055b5f0a2bd49a91d3: sandbox cd2b06716a4aece8d92d34558e4106334eb7f2b7ff96d6055b5f0a2bd49a91d3 not found"
```
I tried to start the container manually but same error: 
```
docker start f759b270b831187099da3a7416f138256dbed90d362ffd269f47e70bedca871b
Error response from daemon: rpc error: code = 2 desc = "containerd: container did not start before the specified timeout"
Error: failed to start containers: f759b270b831187099da3a7416f138256dbed90d362ffd269f47e70bedca871b
```
Status of all containers in `docker ps -a` are this -- *Exited x hours ago*.

Here is my `docker info`:
```
docker info
Containers: 28
 Running: 0
 Paused: 0
 Stopped: 28
Images: 82
Server Version: 1.11.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 149
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge null host
Kernel Version: 3.19.0-25-generic
Operating System: Ubuntu 14.04.3 LTS
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 3.86 GiB
Docker Root Dir: /var/lib/docker
Debug mode (client): false
Debug mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
```

**Comment by HackToday** [comment]
seems error message like issue https://github.com/docker/docker/issues/22226

How do you hit such issue ? any ways to reproduce that ?

**Comment by devlina** [comment]
Not sure how to reproduce that. I am creating kubernetes and docker registry containers. I hit the issue first time few days back when I flushed iptables and restarted docker. At that time I was debugging skydns setup and had several changes applied on, so I reinstalled everything from scratch. After fresh install even flushing out iptables and restarting docker worked absolutely fine. It was working good but started again yesterday. So its hard to tell what triggered this.
